src/eclingo/internal_states/internal_control.py

```diff
@@ -66,6 +66,8 @@ def __init__(
         self.control.register_observer(
             clingox_program.ProgramObserver(self.ground_program)
         )
+        
+        # HERE add the new observer for the Reifier
 
```

Comment:
@jorgefandinno Okay, so my first idea, and the one I was trying to implement was to use the same use_reification boolean value that we have used before, and then register an observer with the Reifier. So, if the value use_reification was used, it will register that observer. However, I thought I could connect that to the AppConfig use_reification, like we did here:
https://github.com/jorgefandinno/eclingo/blob/29d779866a3e9206dd671f050e447796a90e0fc3/src/eclingo/parsing/parser.py#L74

However, are we planning on passing an extra command argument, like when we do echo "a :- b. k{b}." | clingo --output=reify  ? 

Or, should I try to implement it as I thought, if the AppConfig use_reification is set to true, then the Reifier is also registered, but never called?